libmscore/mscore.cpp

```diff
@@ -164,6 +164,7 @@ void Direction::fillComboBox(QComboBox* cb)
       }
 
 static Spatium doubleToSpatium(double d)       { return Spatium(d); }
+static Direction intToDirection(int i)         { return Direction(i); }
```

Comment:
Everywhere we call setDirection(v.value<Direction>()); so in articulations.cpp, tuplet, stem, everywhere.